### ISSUE TITLE
Container element inside symbol inside defs is not rendered when referenced

### ISSUE THREAD

**Issue by Ionaru** [dnfield/flutter_svg#635]
Issue split off from https://github.com/dnfield/flutter_svg/issues/630.

**Description**
Whenever a container element (`<g>`, `<a>`, etc) are nested inside a `<symbol>` which itself is nested in a `<defs>`, that container element (or the contents of the container, difficult to distinguish), are not rendered when referenced through `<use>`.

**SVG code**
```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" width="200" height="200">
   <defs>
      <symbol id="circle">
         <g>
            <circle cx="50" cy="50" r="35" />
         </g>
      </symbol>
   </defs>
   <use href="#circle" />
</svg>
```

**Expected result (Chrome)**
![image](https://user-images.githubusercontent.com/3472373/147028363-d7dffccd-b3a2-48c6-9945-a844ea91d20a.png)

**Actual result (Flutter)**, ignore image size differences.
![image](https://user-images.githubusercontent.com/3472373/147028394-01d5c73d-3d0a-4086-a556-aa7a96cf3dde.png)

**Version info**
- flutter_svg 1.0.0
- Flutter 2.8.0
- Dart 2.15.0
- Chrome 96
- Windows 10

**Comment by dnfield** [comment]
I really appreciate these smaller breakdowns. I agree this is a bug. I'll see if I can further root cause it - seems like some bug in the way symbol is handled.

**Comment by dnfield** [comment]
Yeah, there's a bug in the way that elements in `defs` are handled. Fix incoming.

**Comment by dnfield** [comment]
Fix is in v1.0.1 (specifcially, 38c638fd5b218c1c6537e02d5286a96e7a940543)
